worker/src/lib/rosetta/RosettaWrapper.ts

```diff
@@ -0,0 +1,28 @@
+import { IDatabaseService } from "@helicone/project-rosetta/dist/src/RosettaTypes";
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Database } from "../../../supabase/database.types";
+import { Rosetta } from "@helicone/project-rosetta/dist/src/Rosetta";
+import { RosettaStore } from "./RosettaStore";
+import { Env } from "../..";
+
+export class RosettaWrapper {
```

Comment:
Wait I am confused- why do we need the mapper in the worker too?

Reply:
This is just for the Cron job. V2 I think we should have 2 different types. RosettaMapper and RosettaGenerator